README.md

```diff
@@ -0,0 +1,84 @@
+# wazuh-kubernetes
+Wazuh (3.2) cluster on top of Kubernetes (tested with 1.8.6) with a working simple ELK stack.
+
+## Abstract
+Wazuh best-practices recommend to deploy multiple instances of the Wazuh manager so the manager can support a larger amount of events and can be fault tolerant.
+* `Master` node - intended to expose the Wazuh API, manage agents registration
+* `Client` nodes - intended to receive agents events
+
+Based on that, we are demonstrating that it's possible to deploy the Wazuh cluster in a Kubernetes cluster on top of AWS. Master and client nodes are all behind an internal AWS elastic load balancer, so the events from agents are all dispatched to each nodes available in the cluster. Also, Kubernetes is assuring the cluster to stay up as much as possible.
+
+This repository is using Docker images from the [wazuh-docker](https://github.com/wazuh/wazuh-docker) repository. A simple `kompose convert -f docker-compose.yml` helped a lot to build this Kubernetes example!
+
+## Pre-requisites
+* Kubernetes cluster (tested with v1.8.6) on top of [AWS](https://aws.amazon.com/). [kops](https://github.com/kubernetes/kops) is a nice tool to deploy such cluster.
```

Comment:
"such a cluster"

Reply:
I will replace with `The Kubernetes documentation is recommending [kops](https://kubernetes.io/docs/getting-started-guides/kops/) to install Kubernetes in AWS.`.